File: hbe-tests/projects/TEST-06-multiple-aar/app/src/main/java/com/hbe/test06/MainActivity.java
package com.hbe.test06;
import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView tv = new TextView(this);
        tv.setText("TEST-06-multiple-aar");
        setContentView(tv);
    }
}

File: hbe-tests/projects/TEST-09-kotlin-dsl/app/src/main/kotlin/dev/hbe/test09/MainActivity.kt
package dev.hbe.test09

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val tv = TextView(this).apply {
            text = "Kotlin DSL Works"
            textSize = 24f
            setPadding(32, 32, 32, 32)
        }
        setContentView(
            MaterialButton(this).apply {
                text = "Kotlin DSL Button"
                setOnClickListener { tv.text = "Clicked (Kotlin DSL)" }
            }
        )
    }
}